### PR TITLE
refactor routes, controllers for wizard Step 1 Satellite + temp api authorization

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -21,10 +21,14 @@ gem 'jquery-rails'
...
 gem 'sdoc', '~> 0.4.0', group: :doc
 
-
 gem 'passenger'
 gem 'rack-cors'
 
+gem 'faraday'
+gem 'faraday_middleware'
+
+gem 'figaro'
```
Comment (cfchase) [inline Gemfile]: Can we avoid using figaro?  I'd hate to add this to the list of packages we have to distribute just to use an environment variable.  Is this only used for development environments?

**Comment by isratrade** [comment]
@cfchase, sure, no problem.

**Comment by isratrade** [comment]
@cfchase, updated for review.

**Comment by cfchase** [comment]
ACK

**Pull request merged**